bench.sh

```diff
@@ -0,0 +1,5 @@
+#! /usr/bin/env bash
```

Comment:
If we're forcing bash as the shell, I like to reflect that in the file extension: `bench.bash`.

Reply:
Though this script doesn't require bash, it could easily be `#! /usr/bin/env sh`